src/default-data.js

```diff
@@ -54,13 +54,20 @@ export const defaultData = {
         TUTORIAL_ISLAND: {
             questCompleted: false,
             unlocked: true,
+            questTier: {
+                giantRat: 1,
+            },
```

Comment:
We have access to current enemies killed and the array of quest tiers, what benefit is there in storing the current tier?

Reply:
I considered adding this to the enemies killed but it resulted in much more unreadable code. I figured that this was a cleaner way to do it albeit somewhat duplicated efforts.